docs/Learning-Environment-Design-Agents.md

```diff
@@ -201,6 +266,17 @@ public override void CollectObservations(VectorSensor sensor)
 }
 ```
 
+`ObservableAttribute` has built-in support for enums:
+```csharp
+enum ItemType { Sword, Shield, Bow }
```

Comment:
You disliked `LastItem` so much you got rid of it here :)

Reply:
Because it's not necessary here

Reply:
I'll add a comment about it. It also handles non-sequential enums (which the CollectObservations code does not); there's an example in the unit tests: https://github.com/Unity-Technologies/ml-agents/blob/ff612160f1c391d7492c41a5702008b636639758/com.unity.ml-agents/Tests/Editor/Sensor/ObservableAttributeTests.cs#L13-L18